### ISSUE TITLE
test(robo): create tests for `loadEnv()`

### ISSUE THREAD

**Issue by Pkmmte** [Wave-Play/robo.js#92]
Changeset: none

**Comment by Tejas-Sharma-dev** [comment]
@Pkmmte is this issue still active, since its very old.

**Comment by Pkmmte** [comment]
@Tejas-Sharma-dev Yes, this is still active and we're open to contributions. We haven't gotten around to this.

**Comment by Tejas-Sharma-dev** [comment]
I'm interested to contribute, I'm learning how robo.js works, and making simple bots using the tutorial.
